### ISSUE TITLE
Option to upload file to temporary file directory instead of always in RAM

### ISSUE THREAD

**Issue by mschwartz** [decafjs/decaf#10]
File uploads are currently limited to the size of the JVM; whatever size Java Byte Array can be allocated.

If a file is really big, maybe it should be written to disk and the req.data member should reflect in memory or on disk.

**Comment by mm765** [comment]
Is this implemented already ?
If not, are you still planning to implement it ?

Regards,
Matthias

**Comment by mschwartz** [comment]
It's not implemented.  Do you plan on uploading massive files?

**Comment by mm765** [comment]
Yes, like videos and 3d files, could get up to 1 Gigabyte or even more, will probably not work in ram.

**Comment by mschwartz** [comment]
I will implement file system upload for you.

**Comment by mm765** [comment]
Cool, thanks.

**Comment by mm765** [comment]
Did you have the time to look at this yet ?
